.github/workflows/ci.yml

```
@@ -95,8 +100,8 @@ jobs:
     - name: Install Android Components
       if: matrix.platform.name == 'Android'
       run: |
-        wget -nv https://dl.google.com/android/repository/android-ndk-r24-linux.zip -P $GITHUB_WORKSPACE
-        unzip -qq -d $GITHUB_WORKSPACE android-ndk-r24-linux.zip
+        echo "y" | ${ANDROID_HOME}/tools/bin/sdkmanager --install "build-tools;33.0.2"
+        echo "y" | ${ANDROID_HOME}/tools/bin/sdkmanager --install "ndk;26.1.10909125"
```

```suggestion
        yes | ${ANDROID_HOME}/tools/bin/sdkmanager --install "build-tools;33.0.2"
        yes | ${ANDROID_HOME}/tools/bin/sdkmanager --install "ndk;26.1.10909125"
```
There's a built-in program for spamming `y` at CLI prompts.

---

Apparently there is a reason why `yes` wasn't used here. I tried it but it results in a `yes: standard output: Broken pipe` error.